src/AccountLabel.jsx

```diff
@@ -58,54 +60,59 @@ class AccountLabelAux extends ReactiveComponent {
 	readyRender () {
```

Comment:
I would really recommend one component per file. Maybe use one folder per component, and add sub-components in sub-folders, like https://github.com/paritytech/js-ui/tree/master/src?

This one is less urgent, we can keep it like this for now.